lib/node_modules/@stdlib/stats/base/snanmeanpn/test/test.ndarray.js

```diff
@@ -130,8 +126,7 @@ tape( 'the function supports a negative `stride` parameter', function test( t )
 		NaN   // 0
 	]);
 
-	N = floor( x.length / 2 );
-	v = snanmeanpn( N, x, -2, 6 );
+	v = snanmeanpn( 4, x, -2, 6 );
```

Comment:
```suggestion
	v = snanmeanpn( 5, x, -2, 8 );
```
see the commented indexes which show how the array is being traversed, same for `test.ndarray.native.js`